src/css/main.css

```diff
@@ -0,0 +1,3 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url("https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous");
+@import url("https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css");
```

Comment:
you have already imported in index.html, if not any specific use case , there is no need to import again